aws/resource_aws_launch_template.go

```diff
@@ -47,6 +47,7 @@ func resourceAwsLaunchTemplate() *schema.Resource {
 			"description": {
 				Type:         schema.TypeString,
 				Optional:     true,
+				Default:      "Managed by Terraform",
```

Comment:
We should skip adding our own default here. 👍 Its an old practice and will cause unexpected differences for existing Terraform configurations when added like this.

Reply:
@bflad Thanks. I removed default description